docs/introduction.md

```diff
@@ -0,0 +1,18 @@
+Studio is a tool to develop AsyncAPI documentation. It allows you to visualize the flow, events, publishers, subscribers as you design your API.
+
```

Comment:
Maybe add more context from studio GH docs?

'One place that allows you to develop an AsyncAPI document, validate it, convert it to the latest version, preview the documentation and visualize the events flow.'